exasol_script_languages_developer_sandbox/runtime/ansible/reset_password_tasks.yml

```diff
@@ -0,0 +1,29 @@
+- name: set new default password
+  ansible.builtin.user:
+    name: "{{ansible_user}}"
+    state: present
+    password: "{{ default_vm_password | password_hash('sha512') }}"
+  become: "{{need_sudo}}"
+
+- name: make password expired
+  ansible.builtin.command: "passwd -e {{ansible_user}}"
+  become: "{{need_sudo}}"
+
+- name: Enable SSH password authentication
+  become: true
+  lineinfile:
+    dest: /etc/ssh/sshd_config
+    regexp: '^#?\s*PasswordAuthentication\s'
+    line: 'PasswordAuthentication yes'
```

Comment:
We should add a note to the user guide to disable this for public VM and use ssh public key authentication

Reply:
Hm, let's discuss offline.

Reply:
I created a new ticket to update the user/developer guide: #19